components/checkbox/index.css

```diff
@@ -158,26 +169,185 @@
 		}
 	}
 
-	/* Selected Invalid */
+	.spectrum-Checkbox-checkmark,
+	.spectrum-Checkbox-partialCheckmark {
+		color: var(--highcontrast-checkbox-background-color-default, var(--spectrum-checkbox-checkmark-color));
+		opacity: 0;
+		transform: scale(0);
+
+		transition:
+			opacity var(--mod-checkbox-animation-duration, var(--spectrum-checkbox-animation-duration)) ease-in-out,
+			transform var(--mod-checkbox-animation-duration, var(--spectrum-checkbox-animation-duration)) ease-in-out;
+	}
+
+	.spectrum-Checkbox-partialCheckmark {
+		display: none;
+	}
+
+	.spectrum-Checkbox-label {
+		text-align: start;
+		margin-inline-start: var(--mod-checkbox-text-to-control, var(--spectrum-checkbox-text-to-control));
+		margin-block-start: var(--mod-checkbox-top-to-text, var(--spectrum-checkbox-top-to-text));
+		margin-block-end: var(--mod-checkbox-bottom-to-text, var(--spectrum-checkbox-bottom-to-text));
+
+		font-size: var(--mod-checkbox-font-size, var(--spectrum-checkbox-font-size));
+		font-weight: var(--spectrum-checkbox-text-font-weight);
+		font-style: var(--spectrum-checkbox-text-font-style);
+
+		transition: color var(--mod-checkbox-animation-duration, var(--spectrum-checkbox-animation-duration)) ease-in-out;
+
+		line-height: var(--mod-checkbox-line-height, var(--spectrum-checkbox-line-height));
+	}
+
+	/**
+	 * The input is visually hidden (with opacity), and remains accessible using assistive tech.
+	 * It is still sized and positioned where it would naturally be present so that touch users can find it with haptics.
+	 */
+	.spectrum-Checkbox-input {
+		color: var(--mod-checkbox-control-color-default, var(--spectrum-checkbox-control-color-default));
+
+		/* Change the font styles in all browsers for input. */
+		font-family: inherit;
+		font-size: 100%;
+		line-height: 1.15;
```

Comment:
Really curious about these styles and why exactly we need them, the comments don't really explain it for me really well and/or I don't have a great grasp of how this would behave with various types of assistive technology. But I know these were here before so this may not be an answerable question.